script/deploy/4_VerifyOFT.s.sol

```diff
@@ -128,7 +135,10 @@ contract VerifyL2OFTAdapter is BaseScript, BatchScript {
         for (uint256 i = 0; i < chainIds.length; i++) {
             uint256 chainId = chainIds[i];
             uint32 eid = getEID(chainId);
-            (,, uint256 limit, uint256 window) = l2OFTAdapter.rateLimits(eid);
+
+            ChainDeployment memory chainDeployment = findChainDeployment(chainId);
```

Comment:
this change ensures we are verifying the contract address from the deployment json file